src/lib/components/Modal/ModalHead.jsx

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  RUIContext,
+  withGlobalProps,
+} from '../../provider';
+import styles from './ModalHead.scss';
+
+export const ModalHead = ({
+  closeButtonDisabled,
+  closeButtonRef,
+  id,
+  onClose,
+  title,
+}) => (
+  <div
+    className={styles.root}
+    id={id}
+  >
+    <h3
+      className={styles.title}
+      {...(id && { id: `${id}__title` })}
+    >
+      {title}
+    </h3>
+    {onClose && (
+      <RUIContext.Consumer>
+        {({ translations }) => (
+          <button
```

Comment:
I thought we were going to use composition for this as well.

In other words, I though, that  the usage would be more along the lines of:
```jsx
<ModalHead id="my-modal-head">
  My Modal Title
  |
  <Button
    ref={modalCloseButtonHead}
    onClick={someHandler} 
    label="Close"
  />
</ModalHead>
```

Reply:
Discuss this on Monday as well. We can't just decompose whole modal as it will not work for styles. And button you show in example is not the button we want to use there.